pkg/virt-handler/cmd-client/manager.go

```diff
@@ -0,0 +1,34 @@
+package cmdclient
+
+import v1 "kubevirt.io/api/core/v1"
+
+type ClientManager interface {
+	Create(vmi *v1.VirtualMachineInstance) (LauncherClient, string, error)
+	Socket(vmi *v1.VirtualMachineInstance) (string, error)
+	IsResponsive(socketPath string) bool
+}
+
+type clientManager struct {
+}
+
+func NewClientManager() ClientManager {
```

Comment:
Please consider returning the concrete type.

Reply:
Done, bad habit of mine.